### PR TITLE
Issue 7: Support 'extends' in the SDK

### PR DESCRIPTION
**Pull request by georgschmidtdumont**

### CODE REVIEW

**Comment by eriksven** [comment]
This PR, assumes BAMM in version 2.0.0. Since we first want to perform a release of the SDK for BAMM 1.0.0, we will postpone the merge of this PR until the BAMM SDK for BAMM 1.0.0 has been released.

File: pom.xml
Comment (atextor) [inline pom.xml:72]: As this version is not on Maven Central yet, this is the last thing blocking the PR from merging. This probably needs to be changed to "2.0.0-M1" or similar when the BAMM 2.0.0 Milestone is released.

**Review by atextor — CHANGES_REQUESTED** [review]
LGTM, the last thing missing is the BAMM version that needs to be set to the correct 2.0.0-M1 or 2.0.0-RC1 version before merging.

**Review by atextor — APPROVED** [review]

**Pull request merged**
